lab_05/readme.md

```diff
@@ -0,0 +1,7 @@
+# LAB-05 Twitter schema uitbreiden met user
+
+-   Breid je huidige schema aan met "user", zorg ervoor dat je je versienummer aanpast
+-   Pas je producer/consumer aan voor het nieuwe schema
+-   Laten ervaren forward/bacward compatibility opties
+-   @Laszlo kun jij dit lab in enkele stappen beschrijven, gezien je ervaring??
```

Comment:
Heb er niet meer hands-on ervaring mee dan jullie, we hebben dit nog niet geïmplementeerd. Maar wil het wel beschrijven.